File: Basic/Java/src/samples/SdsBoundaryType.java
package samples;

public enum SdsBoundaryType {

    Exact(0),
    Inside(1),
    Outside(2),
    ExactOrCalculated(3);

    private final int SdsBoundaryType;

    private SdsBoundaryType(int id) {
        this.SdsBoundaryType = id;
    }

    public int getValue() {
        return SdsBoundaryType;
    }
}

File: ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/SdsBoundaryType.cs
// <copyright file="SdsBoundaryType.cs" company="OSIsoft, LLC">
//
// </copyright>

using System;
using System.Collections.Generic;
using System.Text;

namespace SdsRestApiCore
{
    public enum SdsBoundaryType
    {
        Exact = 0,
        Inside = 1,
        Outside = 2,
        ExactOrCalculated = 3
    }
}
